src/Engine/ExpressionInterface.php

```diff
@@ -114,4 +114,25 @@ public function getUuid();
    */
   public function setUuid($uuid);
 
+  /**
+   * Prepares the execution metadata state.
```

Comment:
this docblock misses information on what "preparing" actually prepares. why would I use this method?